plugins/src/android/OptimoveInitProvider.java

```diff
@@ -74,4 +92,12 @@ private String getStringConfigValue(String packageName, Resources resources, Str
 
         return resources.getString(resId);
     }
+
+    @Nullable
+    private String handleNullValues(String credentials) {
+        if (credentials.equals("undefined") || credentials.equals("null") || credentials.isEmpty()) {
```

Comment:
Can we refactor so the file writes empty string values then perhaps handle with TextUtils.isNullOrEmpty() ?

Reply:
sure, done :)